packages/ra-ui-materialui/src/input/SearchInput.tsx

```diff
@@ -12,7 +12,7 @@ import TextInput from './TextInput';
 const useStyles = makeStyles(
     {
         input: {
-            marginTop: 32,
+            // marginTop: 32,
```

Comment:
Why did you comment this? Does not seem related to the issue. Besides, we don't keep commented out code

Reply:
i removed this unrelated code changes now in latest commit